src/algebra/module/basic.lean

```diff
@@ -99,6 +99,68 @@ protected def function.surjective.module [add_comm_monoid M₂] [has_scalar R M
   zero_smul := λ x, by { rcases hf x with ⟨x, rfl⟩, simp only [← f.map_zero, ← smul, zero_smul] },
   .. hf.distrib_mul_action f smul }
 
+/-- push forward the action of `R` on `M` along a compatible surjective map `f : R →+* S`. -/
+@[reducible]
+def function.surjective.module_left {R S M : Type*} [semiring R] [add_comm_monoid M]
+  [module R M] [semiring S] [has_scalar S M]
+  {f : R →+* S} (hf : function.surjective f) (hsmul : ∀ c (x : M), f c • x = c • x) :
```

Comment:
```suggestion
  (f : R →+* S) (hf : function.surjective f) (hsmul : ∀ c (x : M), f c • x = c • x) :
```
to match `function.surjective.module`; often `hf` is a a proof of `surjective unbundled_f` not `surjective ⇑f`, and making both arguments explicit makes this easier to handle.